### PR TITLE
opacity according to number of voters

### PR DESCRIPTION
**Pull request by salman-kh**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,7 +14,21 @@
     <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
     <script src="http://code.jquerygeo.com/jquery.geo-1.0.0-rc1.min.js"></script>
     <script src="js/leaflet-sidebar.min.js"></script>
+     
 </head>
+    
+<style>
+    
+body{
```
Comment (JanVan01) [inline index.html]: Should be in the style.css file

File: js/data.js [hunk 2]
```diff
@@ -51,41 +51,30 @@ function sparql2GeoJSON(input) {
...
 
+
 function reservesStyle(feature) {
    return {
        fillColor: feature.properties.no > feature.properties.yes ? 'red' : 'green',
        weight: 2,
        opacity: 1,
        color: 'white',
-       fillOpacity: 0.7
+       fillOpacity: getOpacity(feature.properties.totalVoters)
```
Comment (JanVan01) [inline js/data.js]: Opacity should be based on the fraction of people that voted. So it should be something like: (yes+no)/totalVoters

File: js/data.js [hunk 3]
```diff
@@ -51,41 +51,30 @@ function sparql2GeoJSON(input) {
...
-    var layer = feature.target;
-    layer.setStyle({
-        weight : 2,
-        color : 'black',
-        fillOpacity : 0.6
-    }
-    );
-    layer.bringToFront();
-}
+function getOpacity(voters) {
```
Comment (JanVan01) [inline js/data.js]: See comment above. If we already have a fraction, we can just set the opacity as that fraction.

**Review by JanVan01 — CHANGES_REQUESTED** [review]

**Review by JanVan01 — APPROVED** [review]

**Review by salman-kh — COMMENTED** [review]
Looks good

**Pull request merged**